extensions/vscode/snippets/dart.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
why did we rename this file to `dart.json`? We will need to update the `package.json` to point to the new snippets file. I would prefer to leave the file name as it was unless there's a specific reason for the rename.

Reply:
Previously (when the file was named `snippets.json`), all snippets needed to specify a scope (`"scope": "dart"`). This is no longer required when renaming the file to `dart.json`.

Now I see `"language": "dart"` is also specified in the `package.json`. Although, I'm not sure if this works the same way when keeping `snippets.json` and removing the scopes from the individual snippets.